src/csharp/Microsoft.Spark.E2ETest/Resources/people_types.json

```diff
@@ -0,0 +1,3 @@
+{ "name": "Michael", "ages": [ 19 ], "cars": [ "Ford", "BMW" ], "info": { "city": "Burdwan", "state": "Paschimbanga" } }
+{ "name": "Andy", "ages": [ 19, 30 ], "age": 30, "cars": [ "Jeep", "Audi" ], "info": { "city": "Los Angeles", "state": "California" } }
+{ "name": "Justin", "ages": [ 19, 30, 40 ], "age": 19, "cars": [ "BMW", "Fiat" ], "info": { "city": "Seattle" } }

```

Comment:
@imback82 @rapoth Are we allowed to mention brand names here?  I'm not sure how the licensing works for that.

Reply:
Good point. If so, I can change to other things.

Reply:
I think it's ok since we are not advertising them.

Reply:
Can we just update people.json with the new keys/values ?

Reply:
I can update people.json with new keys/values. But we are testing `int[]` using ages and show `ArrayList`  example using cars.